app/models/nonprofit.rb

```diff
@@ -107,6 +107,7 @@ def build_admin(**kwargs)
   validate :user_is_valid, on: :create, unless: -> {register_np_only}
   validate :user_registerable_as_admin, on: :create, unless: -> {register_np_only}
   validate :timezone_is_valid
+  validate :state_in_us?, on: %i(create update)
```

Comment:
There's no real gain setting the "on" argument to `create` and `update` so it should be removed. The default is to validate before every save and the two "types" of saves are `create` and `update` so mentioning them both is redundant.

Reply:
Okay, we'll fix it. Thanks for the explanation!